README.md

```diff
@@ -302,22 +326,136 @@ RUN xx-go build -o hello ./hello.go && \
 
 If you want to make `go` compiler cross-compile by default, you can use `xx-go --wrap` and `xx-go --unwrap`
 
-```
-...
+```dockerfile
+# ...
 RUN xx-go --wrap
 RUN go build -o hello hello.go && \
     xx-verify hello
 ```
 
-### External SDK support
+## Rust
+
+Building Rust can be achieved with the `xx-cargo` wrapper that automatically
+sets up the target triple and also `pkg-config` and C compiler.
+
+The wrapper supports rust installed via [`rustup`](https://rustup.rs/)
+(alpine/debian), distribution packages (alpine/debian) and the [official `rust` image](https://hub.docker.com/_/rust).
+
+### Building on Alpine
+
+```dockerfile
+# syntax=docker/dockerfile:1
+# official rust image
+FROM --platform=$BUILDPLATFORM rust:alpine
+RUN apk add clang lld
+# ...
+RUN --mount=type=cache,target=/usr/local/cargo/git/db \
+    --mount=type=cache,target=/usr/local/cargo/registry/cache \
+    --mount=type=cache,target=/usr/local/cargo/registry/index \
+    cargo fetch
+ARG TARGETPLATFORM
+RUN xx-cargo build --release --target-dir ./build && \
```

Comment:
We should show/document the `xx-cargo --setup-target-triple`. It is optional but usually recommended because having it on a separate layer will make sure that it is cached separately.